packages/superset-ui-core/test/query/buildQueryObject.test.ts

```diff
@@ -47,30 +55,34 @@ describe('buildQueryObject', () => {
       metric: 'sum__num',
       secondary_metric: 'avg__num',
     });
-    expect(query.metrics).toEqual([{ label: 'sum__num' }, { label: 'avg__num' }]);
+    expect(query.metrics).toEqual(['sum__num', 'avg__num']);
   });
 
   it('should group custom metric control', () => {
-    query = buildQueryObject({
-      datasource: '5__table',
-      granularity_sqla: 'ds',
-      viz_type: 'table',
-      my_custom_metric_control: 'sum__num',
-      queryFields: { my_custom_metric_control: 'metrics' },
-    });
-    expect(query.metrics).toEqual([{ label: 'sum__num' }]);
+    query = buildQueryObject(
+      {
+        datasource: '5__table',
+        granularity_sqla: 'ds',
+        viz_type: 'table',
+        my_custom_metric_control: 'sum__num',
+      },
+      { my_custom_metric_control: 'metrics' },
+    );
+    expect(query.metrics).toEqual(['sum__num']);
   });
 
   it('should group custom metric control with predefined metrics', () => {
-    query = buildQueryObject({
-      datasource: '5__table',
-      granularity_sqla: 'ds',
-      viz_type: 'table',
-      metrics: ['sum__num'],
-      my_custom_metric_control: 'avg__num',
-      queryFields: { my_custom_metric_control: 'metrics' },
-    });
-    expect(query.metrics).toEqual([{ label: 'sum__num' }, { label: 'avg__num' }]);
+    query = buildQueryObject(
+      {
+        datasource: '5__table',
+        granularity_sqla: 'ds',
+        viz_type: 'table',
+        metrics: ['sum__num'],
+        my_custom_metric_control: 'avg__num',
+      },
+      { my_custom_metric_control: 'metrics' },
+    );
+    expect(query.metrics).toEqual(['sum__num', 'avg__num']);
```

Comment:
I wish GH could show a proper diff on this

Reply:
![image](https://user-images.githubusercontent.com/335541/102677563-e717f800-4157-11eb-8b93-3e96e9b3360e.png)

You mean like this?

Reply:
Live and learn! 👍